tests/unit_test/data_processor/data_processor_test.py

```
@@ -15981,6 +15981,23 @@ def test_add_email_action_with_custom_text(self):
+                        "smtp_url": "test.test.com",
+                        "smtp_port": 25,
+                        "smtp_userid": None,
+                        "smtp_password": {'value': "test"},
+                        "from_email": {"value": "from_email", "parameter_type": "slot"},
+                        "to_email": {"value": ["test@test.com", "test1@test.com"], "parameter_type": "value"},
+                        "subject": "Test Subject",
+                        "response": "Test Response",
+                        "tls": False,
+                        "custom_text": {"value": "Hello from kairon!"}
+                        }
+        with patch("kairon.shared.utils.SMTP", autospec=True) as mock_smtp:
```

Ruff (0.8.2)

Local variable `mock_smtp` is assigned to but never used

Remove assignment to unused variable `mock_smtp`

(F841)